src/clj/kixi/hecuba/data/measurements.clj

```diff
@@ -210,14 +210,24 @@
                              nil))))
        measurements))
 
+(defn where
+  "Takes a map of clauses for database query and returns
+  a vector of vectors, where :start and :end keys are
+  renamed to :timestamp. Follows Hayt rules."
+  [m]
+  (mapv (fn [[k v :as item]] (case k
+                     :start (into [>= :timestamp] [v])
```

Comment:
why isn't this just `start [>= :timestamp v]` 

Am I missing something? same for the following `into` calls?